javatests/google/registry/rde/imports/RdeImportUtilsTest.java

```diff
@@ -121,14 +132,20 @@ public void testImportNewContact() {
   public void testImportExistingContact() {
     ContactResource newContact = buildNewContact();
     persistResource(newContact);
-    ContactResource updatedContact =
+    final ContactResource updatedContact =
         newContact
             .asBuilder()
             .setLastEppUpdateTime(newContact.getLastEppUpdateTime().plusSeconds(1))
             .build();
-    assertThat(rdeImportUtils.importContact(updatedContact)).isFalse();
-
-    // verify the updated contact was saved
+    try {
+      ofy().transact(new VoidWork() {
+        public void vrun() {
```

Comment:
I'm adding @Override to a lot of the vrun methods in this file (they need them).